library/std/src/process.rs

```diff
@@ -90,8 +90,8 @@
 //!
 //! # Windows argument splitting
 //!
-//! On Unix systems arguments are passed to a new process as an array of strings
-//! but on Windows arguments are passed as a single commandline string and it's
+//! On Unix systems arguments are passed to a new process as an array of strings,
+//! but on Windows arguments are passed as a single commandline string and it is
```

Comment:
Commander Data, is that you?

Reply:
Lol, the grammar lords always said to avoid short contractions in technical writing